components/Navbar.js

```diff
@@ -80,12 +80,12 @@ function Navbar() {
 
         {/* other nav links */}
         <div className="hidden lg:flex items-center space-x-3 ">
-          <Link href="/register">
+          <Link href="/Signup">
```

Comment:
please keep this as /register and rename the file to lowercase only using `git mv Signup.js register.js` then commit